packages/story-editor/src/components/floatingMenu/elements/layerOpacity.js

```diff
@@ -23,14 +23,30 @@ import { __ } from '@googleforcreators/i18n';
 /**
  * Internal dependencies
  */
-import { IconButton } from './shared';
+import { useStory } from '../../../app';
+import { Input, useProperties } from './shared';
 
 function LayerOpacity() {
+  const { opacity } = useProperties(['opacity']);
+  const updateSelectedElements = useStory(
+    (state) => state.actions.updateSelectedElements
+  );
+
+  const handleOpacityChange = (_, value) =>
+    updateSelectedElements({
```

Comment:
Could we also add tracking? An example of what exists for floating menu:
```
    trackEvent('floating_menu', {
        name: 'set_text_color',
        element: 'text',
      });
```

We also have an issue for implementing tracking (#10871) where it was missed but since it's still an open PR, mb we can add it now already.